File: apps/android/app/src/main/kotlin/dev/steeringphone/protocol/PacketConstants.kt
package dev.steeringphone.protocol

object PacketConstants {
    const val MAGIC: Byte = 0xD5.toByte()
    const val VERSION: Byte = 0x01
    const val PACKET_SIZE: Int = 43
    const val DISCOVERY_PORT: Int = 45678
    const val WEBSOCKET_PORT: Int = 45679
    const val UDP_DATA_PORT: Int = 45680
    const val HEARTBEAT_INTERVAL_MS: Long = 1000L
    const val CONNECTION_TIMEOUT_MS: Long = 3000L
}

File: apps/windows/SteeringPhone.Core/Protocol/PacketConstants.cs
namespace SteeringPhone.Core.Protocol;

public static class PacketConstants
{
    public const byte Magic = 0xD5;
    public const byte Version = 0x01;
    public const int PacketSize = 43;
    public const int DiscoveryPort = 45678;
    public const int WebSocketPort = 45679;
    public const int UdpDataPort = 45680;
    public const int DefaultUdpPort = UdpDataPort;
    public const int DEFAULT_UDP_PORT = UdpDataPort;
    public const long HeartbeatIntervalMs = 1000;
    public const long ConnectionTimeoutMs = 3000;
}
